web/src/assets/css/common.css

```diff
@@ -189,13 +189,13 @@
     float: right;
 }
 .date-select-32 {
-    padding: 0px 10px !important;
+    padding: 0 10px !important;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of !important](https://app.codacy.com/gh/WeBankFinTech/WeBASE-Front/pullRequest?prid=5240357)